nix/pkgs/nim/default.nix

```diff
@@ -0,0 +1,240 @@
+# https://nim-lang.github.io/Nim/packaging.html
```

Comment:
Maybe you can add a comment explaining why this file is here?
Or better, add a `nix/pkgs/nim/README.md` explaining why this derivation is here instead of the normal package.